app/migrations/versions/180aa40ff01e_.py

```diff
@@ -0,0 +1,24 @@
+"""empty message
```

Comment:
I never worked with these migrations files before. The readme specifcally mentioned to commit these however I ran `python manage.py db migrate`. Which may resulted in multiple migration files?? Is that okay?

Reply:
you might have run migrate multiple times when nothing changed. you only want to commit the empty ones if it merges the head when it has branched off, which is not the case here. if you delete all the new migrations and run migrate again, it should just generate one clean migration

Reply:
This migration file is pretty deep in the history, so I'm just gonna leave it - it's not really causing any harm anyways lol

```
46903aca6fe1 -> 8a2e011771d6 (head), empty message
a487024cdea9 -> 46903aca6fe1, empty message
8aa8a0606156 -> a487024cdea9, empty message
180aa40ff01e -> 8aa8a0606156, empty message
1a8a9213e85d -> 180aa40ff01e, empty message
1a8a9213e85d -> af906681b8b2 (head), empty message
132ea53022f1 -> 1a8a9213e85d (branchpoint), empty message
5c5b65647bed -> 132ea53022f1, empty message
ae98d36420a1 -> 5c5b65647bed, empty message

```

Reply:
1a8a9213e85d and a487024cdea9 are identical and causing migrations to break, while 180aa40ff01e and 8aa8a0606156 are empty:

`46903aca6fe1 -> 8a2e011771d6 (head),
a487024cdea9 -> 46903aca6fe1, empty message
8aa8a0606156 -> a487024cdea9, BREAKS
180aa40ff01e -> 8aa8a0606156, NULL
1a8a9213e85d -> 180aa40ff01e, NULL
132ea53022f1 -> 1a8a9213e85d (branchpoint), empty message
`

so gonna manually re-route 1a8a9213e85d  to 46903aca6fe1...

Reply:
hm, won't deleting all the migrations from this PR and then just generating one new one fix it?

Reply:
yeah, but then u 2 would have to mess with ur db version. probably would have been smarter actually

Reply:
So I did this: 
- fetch your commits
- `git reset --hard upstream/ruben/uss`
- changed the user model : bio & gender --> enrolled_by & referred_by

How do I now migrate to the new version?

Reply:
k i bombed all the migrations and created just one new one. @RubenGeo you'll probably have to drop your database and recreate it now